resources/Info-merge.plist

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These were simply additional, required plist entries that were needed for the 64 bit version to run properly?

Reply:
For the 32-bit build, we just copy the `Info.plist` we have in `resources` to the executable. For the 64-bit build, `AppBundler` generates an `Info.plist` that doesn't contain everything we need, so we add the items we need that aren't generated by AppBundler, which is a subset of what's in our original `Info.plist`.

Reply:
Got it.  Makes sense.